pkg/preference/preference.go

```diff
@@ -180,9 +192,124 @@ func NewPreferenceInfo() (*PreferenceInfo, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if c.OdoSettings.Experimental != nil && *c.OdoSettings.Experimental {
+		if c.OdoSettings.RegistryList == nil {
+			// Handle user has preference file but doesn't use dynamic registry before
+			defaultRegistryList := []Registry{
+				{
+					Name: "CheDevfileRegistry",
```

Comment:
const

Reply:
All below needs to be a const. Maybe it could be a const of []Registry and then here with `defaultRegistryList` you can append to it?

Reply:
In think in Go, constants can only be scalar values which are character, string, boolean, or numeric values. If we use constant here we should define multiple different constants for `Name` and `URL` for different registries instead of assigning values to the share field directly given those values are only used in this code block, IMO converting to constant here is not good for readability and maintainability. Let me know what you think.

Reply:
Can we make the values in the default registry list constants instead?
So `CheDevfileRegistry`, `https://che-devfile-registry.openshift.io`, `DefaultDevfileRegistry`, `https://raw.githubusercontent.com/elsony/devfile-registry/master`. Having them be constants will make it easier to update the URLs when we need to in the future.

Reply:
That's what I mention above, we have to define multiple different constants for `Name` and `URL` given only this code block needs the those default values, but I can make the change if you think those values are useful in the future.